data/bigquery/v0/insert.go

```diff
@@ -31,8 +32,10 @@ func insertDataToBigQuery(projectID, datasetID, tableName string, valueSaver Dat
 func getDataSaver(input *structpb.Struct, schema bigquery.Schema) (DataSaver, error) {
 	inputObj := input.GetFields()["data"].GetStructValue()
 	dataMap := map[string]bigquery.Value{}
+	transformer := base.InstillDynamicFormatTransformer{}
```

Comment:
@chuang8511
I don't think we need to convert the case here. Since the schema is not defined in VDP, let's just keep it the same as it is in BigQuery.

Reply:
Yeah, I know. But what if in BigQuery, there are two columns `fooBar` and `foo_bar` at the same time? The logic will go wrong.

Reply:
So, do you mean that we do not have to transform BigQuery schema into kebab-case for VDP schema?
If so, I will revert this part & [this code](https://github.com/instill-ai/component/pull/156/files#diff-b54962b9f02adac8f1a743deac5ab5f543d79411a4b1c09578d71902aaa6e8a9R266)

Reply:
Yes, I think we can just use the exact same names as those in BigQuery.